packages/@angular/cli/models/config/config.ts

```diff
@@ -61,6 +61,12 @@ export class CliConfig<JsonType> {
     this._config.$$set(jsonPath, value);
   }
 
+  public getPaths(baseJsonPath: string, keys: string[]) {
+    const ret: { [k: string]: any } = {};
+    keys.forEach(key => ret[key] = this.get(`${baseJsonPath}.${key}`));
```

Comment:
I actually spent a fair bit of time figure out what the best way to do this function was, I was torn between the reduce and forEach. The reduce also needs a typing for the accumulator:

```
    return keys.reduce((acc, key) => {
      acc[key] = this.get(`${baseJsonPath}.${key}`);
      return acc;
    }, {} as { [k: string]: any });
```

All the same to me though, I left the forEach for now but will change if there's a preference.